FlowCal/plot.py

```diff
@@ -590,15 +576,15 @@ def scatter2d(data_list,
     elif hasattr(data_plot, 'channels'):
         plt.ylabel(data_plot.channels[1])
 
-    # Set plot limits if specified, else extract range from domain
+    # Set plot limits if specified, else extract range from range
```

Comment:
this comment doesn't actually make a lot of sense now that you've swapped "domain" for "range", lol